src/modules/position_estimator_inav/position_estimator_inav_main.c

```diff
@@ -192,6 +190,7 @@ static void write_debug_log(const char *msg, float dt, float x_est[2], float y_e
 	float corr_vision[3][2], float w_xy_vision_p, float w_z_vision_p, float w_xy_vision_v)
 {
 	return;
+#ifndef __PX4_QURT
```

Comment:
I'd like to push back on this one - the codebase is suffering increasingly from portability issues since we have massive amounts of conditional compilation. I think its much worse to have platform-specific switches than to trim functionality or guard this with an #ifdef INAV_DEBUG. This applies in general - the application layer should not have *any* platform specific IFDEFs and I'm happy to change the application layer to enable this.